src/utils/locales/fr.js

```diff
@@ -86,9 +86,11 @@ export default {
     frisbeeTab: 'Frisbee',
     fitnessTab: 'Fitness',
     theoryTab: 'Théorie',
+    teamTab: 'Mon Équipe',
     editor: 'Éditeur',
     drills: 'Exercices',
-    programs: "Programmes d'entraînement",
+    programs: 'Programmes pour Adultes',
```

Comment:
Maybe we should use subtitle just like we did on fitness?

Reply:
I have the feeling that it's clear enough like it